generators/aws-vpc/index.js

```diff
@@ -8,6 +8,18 @@ export default class AWSVPCGenerator extends PulumiGenerator {
 
     this.displayName = "AWS VPC";
     this.name = "aws-vpc";
+
+    this.option("projectName", {
+      default: this._getDefaultProjectName(),
```

Comment:
I don't think an option should have a default value.
Especially when there is a prop for that same setting anyway. The prop should have the default, not the option.

Reply:
Isn't the `projectName` option inherited from the PulumiGenerator? Why is it redefined here?